addons/mail/static/src/components/message/message.js

```diff
@@ -94,7 +89,7 @@ export class Message extends Component {
     get isActive() {
         return Boolean(
             this.state.isHovered ||
-            this.state.isClicked ||
+            (this.messageView && this.messageView.isClicked) ||
```

Comment:
When both `isClicked` and `isHovered` are merge, this condition will be really messy with multiple guard on `this.messageView`.
I think this could be more readable/mergable if it was refactored into multiple `if` statement.
What do you think ?